movies/models.py

```diff
@@ -1,26 +1,68 @@
 from django.db import models
 from django.contrib.auth.models import User
 from django.utils.functional import cached_property
-from cinema import settings
-from cinemas.models import Hall
-from datetime import timedelta
+from cinema import settings as base_settings
+from django.core.files.uploadedfile import SimpleUploadedFile
+from PIL import Image
+import io
+import os
 
 # Create your models here.
 
 
 class Movie(models.Model):
     user = models.ForeignKey(User, null=True, on_delete=models.CASCADE)
     name = models.CharField(max_length=100, null=True, verbose_name='Title')
-    poster = models.URLField(verbose_name='Poster')
+    poster = models.ImageField(verbose_name='Poster', max_length=500)
     description = models.TextField(max_length=200, null=True, verbose_name='Description', blank=True)
     imdb_id = models.CharField(max_length=30, null=True, verbose_name='IMDB_id')
     length_min = models.IntegerField(verbose_name='Length')
     trailer_url = models.URLField(max_length=200, null=True, blank=True)
     is_scheduled = models.BooleanField(default=False)
+    thumbnail = models.ImageField(upload_to='thumbnails', max_length=500, null=True, blank=True)
```

Comment:
Maybe we can establish a naming rule, so that we wouldn't need a new field for it. In case we need more dimensions on the website, we can configure multiple dimensions and create thumbnails for each one. We could use the image name as a prefix and add, based on the dimensions, to the name. For the frontend, we could configure a template tag (a python function recognizable by the template) that takes the image and the dimension name (ex: `small`, `medium`, `large`, `cover` etc.) and translates it to the image name.